src/ensembl/src/shared/components/search-field/SearchField.scss

```diff
@@ -1,9 +1,16 @@
 @import 'src/styles/common';
 
 .searchField {
-  border: 1px solid $grey;
+  box-shadow: $form-field-shadow;
+  border: none;
+  padding: 0 15px;
+  height: 30px;
   display: inline-flex;
-  padding: 0 0.6em;
+  padding: 0 12px;
+
+  input {
+    background-color: transparent;
+  }
 }
```

Comment:
Do we need to specify `border` and `padding-right` in `.searchField`? It doesn't seem to make any change to the search field. Same with `background-color` in `input`.

Also noticed that padding is specified twice in `.searchField`.

Reply:
Background-color actually does make a difference:

https://user-images.githubusercontent.com/6834224/131754484-52c82ec8-bfe4-412d-b906-067c0e4d0eeb.mp4

Instead of the background colour, this could potentially be solved with a bit of vertical padding. Don't know if that would be better.

I'm removing the extra padding rule in `.searchField`, as well as the border and padding-right 👍  I also wonder whether `background: $white;` should be moved from the `.speciesSearchField` in `SpeciesSearchField.scss` to the `.searchField` class.